File: TODOList.Domain/Models/ResponseModels/QueryResponseModels/GetAllTaskItemsQueryResponseModel.cs
using System;

namespace TODOList.Domain.Models.ResponseModels.QueryResponseModels
{
    public class GetAllTaskItemsQueryResponseModel
    {
        public GetAllTaskItemsQueryResponseModel(string id, string name, DateTime createDate, bool isDone)
        {
            Id = id;
            Name = name;
            CreateDate = createDate;
            IsDone = isDone;
        }

        public string Id { get; private set; }
        public string Name { get; private set; }
        public DateTime CreateDate { get; private set; }
        public bool IsDone { get; private set; }
    }
}

File: frontend/src/app/models/response/query/getAllTaskItemsQueryResponseModel.ts
export default class GetAllTaskItemsQueryResponseModel {
    constructor(id: string, name: string, createDate: Date, isDone: boolean) {
        this.id = id
        this.name = name
        this.createDate = createDate
        this.isDone = isDone
    }

    id: string
    name: string
    createDate: Date
    isDone: boolean
}
